Source/WebKit/_WebKit_SwiftUI/API/View+WebViewModifiers.swift

```diff
@@ -30,50 +30,56 @@ extension View {
     @available(WK_IOS_TBA, WK_MAC_TBA, WK_XROS_TBA, *)
     @available(watchOS, unavailable)
     @available(tvOS, unavailable)
-    public func webViewBackForwardNavigationGestures(_ value: WebView.BackForwardNavigationGesturesBehavior = .automatic) -> some View {
+    public nonisolated func webViewBackForwardNavigationGestures(_ value: WebView.BackForwardNavigationGesturesBehavior) -> some View {
         environment(\.webViewAllowsBackForwardNavigationGestures, value)
     }
 
-    @_spi(Private)
-    public func webViewMagnificationGestures(_ value: WebView.MagnificationGesturesBehavior) -> some View {
+    /// Determines whether magnify gestures change the view’s magnification.
+    @available(WK_IOS_TBA, WK_MAC_TBA, WK_XROS_TBA, *)
+    @available(watchOS, unavailable)
+    @available(tvOS, unavailable)
+    public nonisolated func webViewMagnificationGestures(_ value: WebView.MagnificationGesturesBehavior) -> some View {
         environment(\.webViewMagnificationGestures, value)
     }
 
     /// Determines whether pressing a link displays a preview of the destination for the link.
     @available(WK_IOS_TBA, WK_MAC_TBA, WK_XROS_TBA, *)
     @available(watchOS, unavailable)
     @available(tvOS, unavailable)
-    public func webViewLinkPreviews(_ value: WebView.LinkPreviewBehavior = .automatic) -> some View {
+    public nonisolated func webViewLinkPreviews(_ value: WebView.LinkPreviewBehavior) -> some View {
         environment(\.webViewAllowsLinkPreview, value)
     }
 
     @_spi(Private)
-    public func webViewTextSelection<S>(_ selectability: S) -> some View where S : TextSelectability {
+    public nonisolated func webViewTextSelection<S>(_ selectability: S) -> some View where S : TextSelectability {
```

Comment:
Nit, you can write this more concisely with `some` types:
```swift
public nonisolated func webViewTextSelection(_ selectability: some TextSelectability) -> some View {
```

This is 100% ABI compatible too, and you can go back and forth between the two spellings (for `some`, not necessarily `any`). Generally I’d prefer this to make reading simpler, unless you need to refer to the metatype within the method body. Kinda technically you can still use `type(of:)` to get it back, though that’s then the dynamic type which can differ in some edge cases too.

Reply:
I thought about that, but don't I need `S` so I can do `S.allowsSelection`?

Reply:
Oh I missed that it was a static requirement, my bad!